meta-dts-distro/recipes-dts/dts/dasharo-deploy/dasharo-deploy

```diff
@@ -552,6 +552,15 @@ update() {
 
   handle_fw_switching $_can_switch_to_heads
 
+  if [ "$UPDATE_VERSION" == "1.1.1" ]; then
+    cpu_gen_check=$(lscpu | grep -F "Model name" | grep -E "(13|14)[0-9]{3}" | wc -l)
```

Comment:
`grep -E "(13|14)[0-9]{3}" ` not sure if this will work as intended. The CPU model number starts with `13XXX` or `14XXX` but why do we catch up to 3 digits if we need only two?

Reply:
@miczyg1 my inital thought was that since its either `13XXX` or `14XXX` then we can check exactly for that, and it worked on my machine. However decreasing the number of trailing numbers from 3 to 2 will work fine too, so i changed it [here](https://github.com/Dasharo/meta-dts/pull/108/commits/8607f9b0b2bfc1640d58c70b1e75883477e23f0f#diff-ac1fdd7af29e82bf8d7ca23462952d638564fce3189e36cbb6eebeb48f480d20R561)